index.js

```diff
@@ -214,7 +214,7 @@ function typeHasher(options, writeTo, context){
         return write(object);
       }
 
-      if(objType !== 'object' && objType !== 'function' && objType !== 'asyncfunction') {
+      if(objType !== 'object' && objType !== 'function' && objType !== 'asyncfunction' && objType !== 'file') {
         if(this['_' + objType]) {
           this['_' + objType](object);
```

Comment:
Is there any particular reason for side-stepping the regular dispatch mechanism using the underscore-prefixed methods in this file?

Reply:
Not really, I just wasn't payng attention.